mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/ArrangorService.kt

```diff
@@ -17,17 +18,12 @@ class ArrangorService(
         .build()
 
     suspend fun hentArrangornavn(arrangorId: Int): String? {
-        val cahchedArrangornavn = arrangorCache.getIfPresent(arrangorId)
+        return CacheUtils.tryCacheFirstNotNull(arrangorCache, arrangorId) {
+            val arrangor = arenaOrdsProxyClient.hentArbeidsgiver(arrangorId)
+            val virksomhet =
+                arrangor?.virksomhetsnummer?.let { amtEnhetsregisterClient.hentVirksomhet(it.toInt()) }
 
-        if (cahchedArrangornavn != null) return cahchedArrangornavn
-
-        val arrangor = arenaOrdsProxyClient.hentArbeidsgiver(arrangorId)
-        val virksomhet =
-            arrangor?.virksomhetsnummer?.let { amtEnhetsregisterClient.hentVirksomhet(it.toInt()) }
-
-        val arrangornavn = virksomhet?.overordnetEnhetNavn
-        arrangornavn.let { arrangorCache.put(arrangorId, it) }
-
-        return arrangornavn
+            virksomhet?.overordnetEnhetNavn ?: ""
```

Comment:
Denne returnerer nå en tom `""` streng i stedet for `null`, er det bevisst?

Reply:
Den bør nok tilbake til `null`. Jeg fikser.

Reply:
Det er fordi cachen-utilen forventer en verdi og godtar ikke null